Cargo.toml

```diff
@@ -36,6 +36,11 @@ tokio = { version = "1.37.0", features = ["test-util", "full", "sync"] }
 futures = "0.3"
 chrono = "0.4"
 uuid = { version = "1.0", features = ["v4"] }
+eigen-crypto-bls = { git = "https://github.com/Layr-Labs/eigensdk-rs/" ,rev = "62556fdb5701dee5a30ea348e7a311be96ab051f"}
```

Comment:
yes, a bug with the signature production currently, may be related to rust/go issues with encoding as previously discussed with @supernovahs. Alternatively, I suspect there may be a masking issue

Reply:
okay - I don't think we want dependency on `eigen-crypto-bls` lib on this - since this is just an example of using it - maybe there's a way to create a separate examples modules which has it's own deps. 

yea - there's a difference between Go and Rust SDK on generating encrypted JSON. I need to fix it. but that should not create bugs since - cerberus and rust sdk creates same format.

Reply:
https://github.com/RonTuretzky/cerberus-api/tree/bug/inconsistent_signing branch for recreation of the bug

https://www.loom.com/share/217262aa5a2647c6a958df2e9f72087e?sid=d537f0e6-0e9a-40aa-a74f-bd673d89f876 loom of the behavior

Reply:
I think for the sake of completeness, support for the eigensdk-rs makes sense as AVS developers will opt into using that and will expect compatibility with Cerberus...